chall01/tkuumola.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import sys
+
+morse_code = {
```

Comment:
I really appreciate your usage of a dict and your implementation with the letters as keys is well thought!